x-pack/plugins/secops/public/components/page/hosts/hosts_table/index.tsx

```diff
@@ -149,9 +150,7 @@ const getHostsColumns = (): Array<Columns<HostsEdges>> => [
               ) : isNil(get('host.id', node)) ? (
                 <>{hostName}</>
               ) : (
-                <EuiLink href={`#/link-to/hosts/${encodeURIComponent(node.host!.id!)}`}>
-                  {hostName}
-                </EuiLink>
+                <HostDetailsLink hostId={node.host!.id!}>{hostName}</HostDetailsLink>
```

Comment:
nit: Same suggestion as my comment above maybe?
https://github.com/elastic/kibana/pull/32365/files#r263482927

About us minimizing the use of bang to turn off TypeScript checking and instead try to go for pure `null` checks or make more things required.